app/models/reservation.rb

```diff
@@ -9,6 +9,14 @@ class Reservation < ActiveRecord::Base
   include Reservations::RelaySupport
   include Reservations::MovingUp
 
+  CATEGORIES = %w(
```

Comment:
I tried moving it and the validation in ReservationValidations failed. See comment above.